packages/components/src/components/CreateCard/CreateCard.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { Icon, IconNames } from '../Icon';
+import { Stack } from '../Stack';
+import { Text } from '../Text';
+
+type StackProps = React.ComponentProps<typeof Stack>;
+
+type CreateCardProps = {
+  icon: IconNames;
+  label: string;
+} & StackProps;
+
+export const CreateCard: React.FC<CreateCardProps> = ({
+  as = 'button',
+  icon,
+  label,
+  ...props
+}) => {
+  return (
+    <Stack
+      as={as}
+      css={{
+        position: 'relative',
+        overflow: 'hidden',
+        maxWidth: '276px',
+        padding: '24px',
+        flex: 1,
+        justifyContent: 'space-between',
+        border: '1px solid',
+        borderColor: 'transparent',
+        borderRadius: '4px',
+        backgroundColor: '#161616',
+        outline: 'none',
+        textDecoration: 'none',
+        transition: 'background ease-in-out, opacity ease-in-out',
+        transitionDuration: theme => theme.speeds[2],
+        '&:hover, &:active': {
+          backgroundColor: 'card.backgroundHover',
+        },
+        '&:focus-visible': {
+          borderColor: 'focusBorder',
+        },
+      }}
+      direction="vertical"
+      gap={4}
```

Comment:
The gap was to ensure a minimum distance between the icon and the label but I guess it won't be needed. Since I added the full height to the component, I removed the gap.

Reply:
That's fine! Whenever we find out that we need it we can always re-add it.